package.json

```diff
@@ -1,157 +1,81 @@
 {
   "name": "strangeluv",
-  "version": "0.0.0",
-  "description": "How I Learned to Stop Worrying and Love React",
+  "version": "2.0.0",
+  "description": "My webpack project",
```

Comment:
Are we gonna change the desc?

Reply:
Good catch!  I'm thinking it might make sense to clear it out altogether.

Reply:
Gonna go ahead and remove this